src/GravityFormsExtensions.php

```diff
@@ -113,11 +113,34 @@ private function hooks(): void
         add_filter('gform_pre_render', [ $this, 'p4_client_side_gravityforms_prefill' ], 10, 1);
         add_filter('gform_form_post_get_meta', [$this, 'p4_gf_enable_default_meta_settings'], 10, 1);
         add_filter('gform_hubspot_form_object_pre_save_feed', [$this, 'p4_gf_hb_form_object_pre_save_feed'], 10, 1);
+        add_action('gform_after_submission', [$this, 'p4_send_gp_pixel_counter'], 10, 2);
 
         add_action('gform_stripe_fulfillment', [ $this, 'record_fulfillment_entry' ], 10, 2);
         add_action('gform_post_payment_action', [ $this, 'check_stripe_payment_status' ], 10, 2);
     }
 
+    /**
+     * Make an API call to the Greenpeace Pixel Counter.
+     * Increase in one unit the total number of submissions for a counter.
+     *
+     * @param array $form The form setting.
+     * @param array $entry A form entry.
+     *
+     */
+    public function p4_send_gp_pixel_counter(array $entry, array $form): void
+    {
+        if (!$form['p4_gf_counter'] || empty($form['p4_gf_counter'])) {
```

Comment:
`!$form['p4_gf_counter']` doesn't take into account the fact that it might be empty? 🤔